src/models/cats.js

```diff
@@ -0,0 +1,12 @@
+const { Sequelize } = require("sequelize");
+
+module.exports = (Sequelize, DataTypes) => {
```

Comment:
The first parameter here should be in lower case as that is how you refer to it on line 11
```
module.exports = (sequelize, DataTypes) => {
```